### DISCUSSION TITLE
Collecting new Spots, Futures, and Perps

### DISCUSSION THREAD

**Discussion by Pebbl3s** [bmoscon/cryptofeed#773] [Q&A]
Hello @bmoscon ,

I am a university student using your library for a crypto related project.

Is there a way to add a feed to the feed handler when a new derivative becomes available?
I want to collect the limit order books for spots, futures, and perpetuals as and when they appear on an exchange.

Is this possible?
Thanks!

**Comment by bmoscon** [comment]
yes, there is an example in the examples/ directory that shows how a new feed can be added after the main event loop has already started. Basically you'd need to watch for a new symbol to be added and then add a new feed to the running feedhandler
